sdcm/cluster.py

```diff
@@ -585,8 +585,8 @@ def install_scylla_bench(self):
             sb_version = f'heads/{sb_version}'
         self.remoter.sudo(shell_script_cmd(f"""\
             rm -rf /usr/local/go
-            rm -f /tmp/sb_install || true
-            mkdir /tmp/sb_install
+            rm -rf /tmp/sb_install || true
+            mkdir -p /tmp/sb_install
             cd /tmp/sb_install
             curl -Lo go.tar.gz https://storage.googleapis.com/golang/go1.17.4.linux-amd64.tar.gz
```

Comment:
Direct installation of the `s-b` is too old approach. We use docker images.
Latest `s-b` supports `go1.18+`. `1.17` which is used here is not supported by the latest `scylla-bench`.
If not latest one is used then need to check the compatibility.

Reply:
@vponomaryov , the target branch is 2021.1, so old branch, old code, and it doesn't support using the docker, as we have on newer branches

Reply:
I am saying that the `go` version used here may be too old. The go version is specified on the line 591 here.

Reply:
i'm giving it a ride [here](https://jenkins.scylladb.com/job/enterprise-2021.1/job/longevity/job/longevity-10gb-3h-gce-test/88/) and soon we will know if it works